2/contrib/s2i/run

```diff
@@ -78,10 +90,13 @@ function install_plugins() {
     /usr/local/bin/install-plugins.sh "${plugins_file}"
   fi
   
-  if [ "$(ls -A /opt/openshift/plugins 2>/dev/null)" ]; then
-    mkdir -p ${JENKINS_HOME}/plugins
-    echo "Copying $(ls /opt/openshift/plugins | wc -l) Jenkins plugins to ${JENKINS_HOME} ..."
-    cp -r /opt/openshift/plugins/* ${JENKINS_HOME}/plugins/
+  if [ "$(ls /opt/openshift/plugins/*.jpi 2>/dev/null)" ]; then
```

Comment:
Sorry for the additional comment - now below is not *.jpi, should that also be the case here and in the echo line?

Reply:
I made an explicit choice to not do that.  Only go into the block if there are *.jpi, and only take the total number of files the *.jpi ls returns for the echo citing the number of plugins, but when we actually copy, in addition to *.jpi, copy any associated pinned files.

The comment below starting with "NOTE" tried to convey that ... was the text there not sufficient?

Reply:
Got it.  I'll stop poking - sorry :)

Reply:
no worries :-)